Week3/MAKEME.md

```diff
@@ -3,21 +3,33 @@
 ## **Todo list**
 
 1. Practice the concepts
-2. MySQL exercises
-3. Code along
+2. Prep exercises
+3. Database exercises
+4. Code along
 
 ## 1. **Practice the concepts**
 
 Let's do a warming up with some interactive exercises! Let's start with doing lesson 13-18 from the following:
 
 - [SQLBolt](https://sqlbolt.com/lesson/select_queries_introduction)
 
-## 2. **MySQL exercises**
+Then let's practice some MongoDB queries! Start by creating an Atlas account over [here](https://www.mongodb.com/cloud/atlas/lp/try2). Atlas is MongoDB's own cloud service that hosts mongodb databases and luckily they offer a very gracious free tier to play around with. Once you have created an account make sure to create a database cluster which can be your playground for the following:
 
-> You will need to do some research to solve these exercises. All the concepts necessary to solve
-these exercises are NOT covered in the reading material. This is on purpose.
+- [MongoDB practice exercises](https://gist.githubusercontent.com/theRemix/7305403e1ab6fc8674f0/raw/c068ab51e930eb133a9443caa314205a89ef4d61/exercise.md) Note that this is a raw markdown file so you may want to copy it into your editor and save it as `.md` to get syntax highlighting. The live version has comments with answers which you can check out after over [here](https://gist.github.com/theRemix/7305403e1ab6fc8674f0#file-exercise-md).
 
-**Exercise 1 : Normalization**
+## 2. **Prep exercises**
+
+> Prep exercises are exercises that you should work on _before_ the session on Sunday. These are a little more difficult or show an important concept and as such are a great exercise to talk about with your mentor. Have a solution ready by Sunday as you may be asked to show what you did.
+
+Inside your `databases` fork, go to the folder `Week3`. Inside of that folder, navigate to `QA_PREP_EXERCISE.md` file which explains what needs to be done. There will also be some questions at the bottom to think about. Go through them _before_ the session on Sunday as it will be covered then.
```

Comment:
| Go through them _before_ the session on Sunday as it will be covered then.
-> Some students might think that it will be covered anyway, so why study. Instead we could phrase it as
` Answer the questions at the end and send them to your teacher before the class`

Reply:
Talked with Tjebbe about this, we feel this is a personal responsibility of the students to do it. If they don't they will fail at some point and we are confident we will catch when they don't understand the materials. We also don't want to put extra pressure on our Q&A mentors to do the checking for this